### PR TITLE
[dest] refactor dest head

### PR DESCRIPTION
**Pull request by zhby99**
refactor dest head

### CODE REVIEW

File: mmseg/models/decode_heads/dest_head.py [hunk 1]
```diff
@@ -1,53 +1,54 @@
 import torch
 import torch.nn as nn
 
+from mmcv.cnn import ConvModule
 from mmseg.models.builder import HEADS
 from mmseg.models.decode_heads.decode_head import BaseDecodeHead
 
 
 @HEADS.register_module()
 class DestHead(BaseDecodeHead):
```
Comment (ghost) [inline mmseg/models/decode_heads/dest_head.py]: DESTHead? DEST is abbreviation like DW.

**Comment by ghost** [comment]
In configs/dest, we only have b0 to b3, you can add b4 and b5 for completeness

**Comment by zhby99** [comment]
> In configs/dest, we only have b0 to b3, you can add b4 and b5 for completeness

@lean76  The config files for b3 and b4 in open sourced DEST are same. @johnyang-nv might have to look at it. 

https://github.com/NVIDIA/DL4AGX/blob/master/DEST/semseg/dest_simpatt-b3_1024x1024_160k_cityscapes.py
https://github.com/NVIDIA/DL4AGX/blob/master/DEST/semseg/dest_simpatt-b4_1024x1024_160k_cityscapes.py

File: mmseg/models/decode_heads/dest_head.py (hunk 1 above)
Comment (zhby99) [reply mmseg/models/decode_heads/dest_head.py]: this is adopted from our open sourced one, any suggestion for name of this head?
btw, what's DW

Comment (ghost) [reply mmseg/models/decode_heads/dest_head.py]: DW is depth-wise, so DestHead can be DESTHead

**Comment by ghost** [comment]
> > In configs/dest, we only have b0 to b3, you can add b4 and b5 for completeness
> 
> @lean76 The config files for b3 and b4 in open sourced DEST are same. @johnyang-nv might have to look at it.
> 
> https://github.com/NVIDIA/DL4AGX/blob/master/DEST/semseg/dest_simpatt-b3_1024x1024_160k_cityscapes.py https://github.com/NVIDIA/DL4AGX/blob/master/DEST/semseg/dest_simpatt-b4_1024x1024_160k_cityscapes.py

For reference segformer's backbone config is at https://github.com/NVlabs/SegFormer/blob/master/mmseg/models/backbones/mix_transformer.py, the depth of DEST b3 or b4 need to be corrected. @johnyang-nv

**Comment by zhby99** [comment]
```
+---------------+-------+-------+
| Class | IoU | Acc |
+---------------+-------+-------+
| road | 97.25 | 98.31 |
| sidewalk | 78.7 | 90.19 |
| building | 89.78 | 95.25 |
| wall | 49.33 | 58.18 |
| fence | 46.25 | 61.0 |
| pole | 52.45 | 61.34 |
| traffic light | 54.67 | 66.02 |
| traffic sign | 66.9 | 74.31 |
| vegetation | 91.0 | 96.29 |
| terrain | 59.07 | 68.58 |
| sky | 93.94 | 97.88 |
| person | 71.75 | 85.63 |
| rider | 41.2 | 50.91 |
| car | 91.77 | 96.37 |
| truck | 49.95 | 57.72 |
| bus | 57.9 | 71.77 |
| train | 42.77 | 63.29 |
| motorcycle | 36.22 | 42.87 |
| bicycle | 67.74 | 84.14 |
+---------------+-------+-------+
2022-11-19 11:07:56,614 - mmseg - INFO - Summary:
2022-11-19 11:07:56,614 - mmseg - INFO -
+-------+-------+-------+
| aAcc | mIoU | mAcc |
+-------+-------+-------+
| 94.42 | 65.19 | 74.74 |
+-------+-------+-------+
```
Training result for current version

**Pull request merged**